### PR TITLE
UATBC Calendar Case List

### PR DESCRIPTION
**Pull request by wpride**
OK, the UI's not perfect but we're not committed to perfect UI for the first pass so calling it here. In addition to adding the notion of a grid of case tiles, this PR also includes:
- Handling for local install references
- Splitting out case list and case tile models/views
- Some random cleanup and error handling/robustness

cross https://github.com/dimagi/formplayer/pull/120

screen shot 2016-09-07 at 7 00 56 pm

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/menus/menu_view.js [hunk 1]
```diff
@@ -211,12 +266,41 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
...
+    });
+
+    MenuList.GridCaseTileViewItem = MenuList.CaseTileView.extend({
+        tagName: "div",
+        className: "formplayer-request case-tile-grid-item",
+        template: "#case-tile-grid-view-item-template",
+    });
+
+    MenuList.GridCaseTileListView = MenuList.CaseTileListView.extend({
+        initialize: function (options) {
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/menus/menu_view.js]: 'options' is defined but never used (no-unused-vars)

**Comment by benrudolph** [comment]
(ping me when this is ready for review)

**Comment by wpride** [comment]
@benrudolph good to review, test should be fixed

**Comment by benrudolph** [comment]
legit looks good on :check

**Pull request merged**